spec/models/item_season_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe ItemSeason, type: :model do
-  let!(:produce_item) { ProduceItem.create(name: 'Banana', category: 'fruit') }
+  let!(:produce_item) { create(:produce_item) }
   subject { described_class.new(produce_item_id: produce_item.id, month_index: 1, country_code: 'br', id: 1) }
```

Comment:
Hardcoding the `id` will lead to [flaky tests](https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html). Any reason for defining it here?

Reply:
No reason, it was a mistake 😬